README.md

```diff
@@ -1,7 +1,34 @@
-# OOP-school-library
+![](https://img.shields.io/badge/Microverse-blueviolet)
+ #  School Library
 Imagine that you are a librarian of a University, this app allows you to record what books are in the library and who borrows them.
 
-## Authors:
+
+ ## Screenshot
+
+![screenshot](./a.png)
+
+ 
+ ## Built With
+
+ - Ruby
+
+ ## Getting Started
+
+ To get a local copy up and running follow these simple example steps.
+- git clone  `https://github.com/Kakalanp/OOP-school-library.git`
+
+- change to the directory using `cd OOP-school-library`
+
+- open with your prefffered code editor.
```

Comment:
- [OPTIONAL] Please there is a typing error with `preferred`, make sure you correct that.

Reply:
Noted, and thank you!